Monika After Story/game/script-topics.rpy

```diff
@@ -17740,3 +17740,42 @@ label monika_hedonism:
     m 7etd "It's no wonder most people don't follow that belief...{w=0.3}it's too simple, where morality is complicated."
     m 1eud "So it makes sense why Oscar Wilde portrayed hedonism in a bad light."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_influencers", # TODO - or instagram models?
+            category=['socialmedia? TODO'],
+            prompt="Influencers",
+            random=True,
+        )
+    )
+
+# TODO:
+# the follow any models Q should be pm - but which?
+
+label monika_influencers:
+    # TODO: sprite codes
+    m 1eua "Do you follow any models on social media?{fast}{nw}"
+    # TODO: consider adding quanitifable answers - do they follow a lot (like a lifestyle)
+    #   or do they follow a few (like a like/want)?
+    menu:
+        m "Do you follow any models on social media?"
+        "Yes.":
+            pass
+        "No.":
+            pass
+
+    m 7eud "I find the culture around online 'models' so interesting."
+    m "Before the age of social media people would only see these types of models and personalities on television or in magazines."
+    m "Models were hired by big companies and were told exactly what they would wear and do, they never had a say."
+    m 6eua "Nowadays Instagram models are completely in charge of their outfits and what they chose to do!"
+    m "It seems a lot more ethical now if you ask me. Even with the online sex work industry, these models are completely in charge of what type of NSFW content they’re making and how much money they’re making. "
```

Comment:
`m 4eub "It seems a lot more ethical now if you ask me. Even with the online sex work industry, these models are completely in charge of what type of NSFW content they’re making and how much money they’re making. "`